page-visitor-information.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * Template Name: Visitor Information
+ *
+ */
+
+get_header();
+
+$fair_description = IGV_get_option('_igv_visitor_options', '_igv_fair_description');
+$fair_image_id = IGV_get_option('_igv_visitor_options', '_igv_fair_image_id');
+
+$schedule = IGV_get_option('_igv_visitor_options', '_igv_schedule');
+
+$tickets = IGV_get_option('_igv_visitor_options', '_igv_ticket_info');
+
+$venue_name = IGV_get_option('_igv_site_options', '_igv_venue_name');
+$venue_address = IGV_get_option('_igv_visitor_options', '_igv_venue_address');
+
+$how_to_arrive = IGV_get_option('_igv_visitor_options', '_igv_how_to_arrive');
+$venue_map = IGV_get_option('_igv_visitor_options', '_igv_venue_map');
+
+$fair_history = IGV_get_option('_igv_page_options', '_igv_fair_history');
+$fair_history_headline = IGV_get_option('_igv_page_options', '_igv_fair_history_headline');
+$fair_history_image_id = IGV_get_option('_igv_page_options', '_igv_fair_history_image_id');
+
+$organizers = IGV_get_option('_igv_page_options', '_igv_organizers_group');
+?>
+
+<!-- main content -->
+
+<main id="main-content">
+
+  <div <?php post_class(); ?> id="post-<?php the_ID(); ?>">
+
+  <!-- main posts loop -->
+  <section id="visitor-info-intro" class="section">
+    <div class="container">
+
+      <div class="row">
+
+        <div class="col col-l col-l-12">
```

Comment:
this section doesnt match the pdf with the title going above the next row rather than just above the image. but i assume thats by design..